scripts/build/build.mjs

```diff
@@ -150,34 +150,47 @@ async function preparePackage() {
 }
 
 async function run(params) {
-  await execa("rm", ["-rf", "dist"]);
-  await execa("mkdir", ["-p", "dist"]);
-  if (!params.playground) {
-    await execa("mkdir", ["-p", "dist/esm"]);
+  const shouldUseCache = !params.file && !params["purge-cache"];
+  const shouldBuildMetaFiles = !params.playground && !params.file;
```

Comment:
This is actually only used as condition for if we should do `preparePackage`. So what about rename to `shouldPreparePackage`.